django/template/base.py

```diff
@@ -976,7 +976,7 @@ def __init__(self, s):
         self.s = s
 
     def __repr__(self):
-        return force_str("<Text Node: '%s'>" % self.s[:25], 'ascii',
+        return force_str("<Text Node: %r>" % self.s[:25], 'ascii',
```

Comment:
while we are here, I think "Text Node" should be `self.__class__.__name__` instead (this is consistent with other code)